spec.proto

```diff
@@ -0,0 +1,55 @@
+/**
+ * CloudEvent Protobuf Format
+ *
+ * - Required context attributes are explicity represented.
+ * - Optional and Extension context attributes are carried in a map structure.
+ * - Data may be represented as binary, text, or protobuf messages.
+ */
+
+syntax = "proto3";
+
+package io.cloudevents.v1;
+
+import "google/protobuf/any.proto";
+import "google/protobuf/timestamp.proto";
+
+option java_package = "io.cloudevents.v1.proto";
```

Comment:
We should work out what we want in terms of options for other languages... which may partly depend on what level of support we expect in SDKs. For example, for C# I'd expect a NuGet package called `CloudNative.CloudEvents.Protobuf`, so it would be nice to have the `csharp_namespace` option for that here.

Reply:
Hey Jon; I only have working knowledge via Java, if there are other language specific annotations that should be added we should do that earlier rather than later.

Reply:
SGTM. Here's an example with Java, Go, Ruby, PHP and C#. (I chose this API as it has a name that should be capitalized in the middle, for DataCatalog):

https://github.com/googleapis/googleapis/blob/master/google/cloud/datacatalog/v1/common.proto

I'm not sure what would be needed for other languages.